src/frontend/jsonapi/baskets.md

```diff
@@ -120,11 +119,11 @@ If the basket is empty, it will return only the basic basket properties but no p
 
 There are three values that you can update using a PATCH request to the basket:
 
-* order.base.customerid (ID of the customers to show the order in their account history)
+* order.base.customerid (ID of the customer to show the order in their account history)
```

Comment:
If it's "customer" than it must be "... in his account ...". We prefer to use plurals to be gender neutral if possible

Reply:
Yes, I forgot to fix "their". But... hmm... it is only possible to act on ONE customer at a time, right? Otherwise, wouldn't the attribute also have to be called "customerids"?

Reply:
It's the ID of one customer and therefore it's "ID of ...", not "IDs of ..."
Maybe it's better to use singular here